config/environments/production.rb

```diff
@@ -76,4 +76,9 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.middleware.use ExceptionNotification::Rack,
+    idobata: {
```

Comment:
Style/AlignParameters: Align the parameters of a method call if they span more than one line.